From a02573be4252c4a4b7226875346cdabfb67b39ad Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 19:00:15 -0400
Subject: [fs] btrfs: sysfs: don't show reserved incompat feature

Message-id: <1407179109-13714-129-git-send-email-sandeen@redhat.com>
Patchwork-id: 87086
O-Subject: [RHEL7.1 PATCH 128/422] btrfs: sysfs: don't show reserved incompat feature
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: David Sterba <dsterba@suse.cz>

commit 66b4bbd4f5895e4b9da03885c00e42ba64f7038e
Author: David Sterba <dsterba@suse.cz>
Date:   Tue Jan 21 18:56:06 2014 +0100

    btrfs: sysfs: don't show reserved incompat feature

    The COMPRESS_LZOv2 incompat featue is currently not implemented, the bit
    is only reserved, no point to list it in sysfs.

    Signed-off-by: David Sterba <dsterba@suse.cz>
    Signed-off-by: Josef Bacik <jbacik@fb.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/sysfs.c b/fs/btrfs/sysfs.c
index ba94b27..1a89386 100644
--- a/fs/btrfs/sysfs.c
+++ b/fs/btrfs/sysfs.c
@@ -198,7 +198,6 @@ BTRFS_FEAT_ATTR_INCOMPAT(mixed_backref, MIXED_BACKREF);
 BTRFS_FEAT_ATTR_INCOMPAT(default_subvol, DEFAULT_SUBVOL);
 BTRFS_FEAT_ATTR_INCOMPAT(mixed_groups, MIXED_GROUPS);
 BTRFS_FEAT_ATTR_INCOMPAT(compress_lzo, COMPRESS_LZO);
-BTRFS_FEAT_ATTR_INCOMPAT(compress_lzov2, COMPRESS_LZOv2);
 BTRFS_FEAT_ATTR_INCOMPAT(big_metadata, BIG_METADATA);
 BTRFS_FEAT_ATTR_INCOMPAT(extended_iref, EXTENDED_IREF);
 BTRFS_FEAT_ATTR_INCOMPAT(raid56, RAID56);
@@ -209,7 +208,6 @@ static struct attribute *btrfs_supported_feature_attrs[] = {
  BTRFS_FEAT_ATTR_PTR(default_subvol),
  BTRFS_FEAT_ATTR_PTR(mixed_groups),
  BTRFS_FEAT_ATTR_PTR(compress_lzo),
- BTRFS_FEAT_ATTR_PTR(compress_lzov2),
  BTRFS_FEAT_ATTR_PTR(big_metadata),
  BTRFS_FEAT_ATTR_PTR(extended_iref),
  BTRFS_FEAT_ATTR_PTR(raid56),
-- 
1.7.1